### PR TITLE
Add E2E test for frame pointer unwinding

### PR DESCRIPTION
**Pull request by florian-kuebler**
This allows specifying the unwinding method and sampling period
in the E2E tests and run the top down and buttom up tests again
with frame pointer unwinding.

For bottum up view, there results were a bit flaky (as the 5-6
seconds) only contained arround 80 samples. Therefore, I increased
the sampling rate here.

Test: Run E2E tests locally
Bug: http://231442903

### CODE REVIEW

File: contrib/automation_tests/orbit_bottom_up.py [hunk 1]
```diff
@@ -33,7 +34,10 @@ def main(argv):
     test_cases = [
         ConnectToStadiaInstance(),
         FilterAndSelectFirstProcess(process_filter='hello_ggp'),
-        Capture(),
+        LoadSymbols(module_search_string="libc-2.24"),
```
Comment (danielfenner) [inline contrib/automation_tests/orbit_bottom_up.py:37]: This looks a bit suspicious: below you also load the symbols for "hello_ggp". Is this intended?

**Review by danielfenner — APPROVED** [review]
Thanks, lgtm!

File: contrib/automation_tests/orbit_bottom_up.py (hunk 1 above)
Comment (florian-kuebler) [reply contrib/automation_tests/orbit_bottom_up.py:37]: Yes, with frame pointers, we wont get any symbol information. The top-down/bottom-up view e2e tests validate the names of the root functions (clone, start, ioctl), and we need the names of those.

**Comment by florian-kuebler** [comment]
Btw. #3669 needs to be in before merging this.

File: contrib/automation_tests/test_cases/capture_window.py [hunk 2]
```diff
@@ -332,6 +333,38 @@ def _set_capture_options(self, collect_thread_states: bool, collect_system_memor
             logging.info('Toggling "Collect memory usage and page faults information" checkbox')
             collect_system_memory_usage_checkbox.click_input()
 
+        logging.info('Setting sampling period to {}.'.format(sampling_period_ms))
```
Comment (reichlfl) [inline contrib/automation_tests/test_cases/capture_window.py:336]: Nit: I think this method has become way too large and should be broken into smaller methods (e.g. _set_sampling_period, _set_frame_pointer_unwinding and so on).

**Review by reichlfl — APPROVED** [review]
LGTM

File: contrib/automation_tests/test_cases/capture_window.py (hunk 2 above)
Comment (florian-kuebler) [reply contrib/automation_tests/test_cases/capture_window.py:361]: Done.

**Review by reichlfl — APPROVED** [review]
Ah nice, thanks for splitting this up. I didn't go over every single line since I assume they are simply copy & paste, but overall LGTM!

**Pull request merged**